src/selection_stream/selection_stream_processor.ts

```diff
@@ -373,6 +366,26 @@ export class SelectionStreamProcessor
     });
   }
 
+  /**
+   *  ---- PRIVATE ----
+   */
+
+  private _setZone(
+    anchor: AnchorZone,
+    mode: SelectionEvent["mode"] = "overrideSelection"
```

Comment:
I would force to be explicit when calling this method by not providing a default value for `mode`